cloudfront/distribution.go

```diff
@@ -49,3 +50,36 @@ func (c *CloudFront) ListDistributions(ctx context.Context) ([]*cloudfront.Distr
 
 	return distributions, nil
 }
+
+// GetDistributionByName gets a cloudfront distribution by the name (by searching until it finds the matching alias)
+func (c *CloudFront) GetDistributionByName(ctx context.Context, name string) (*cloudfront.DistributionSummary, error) {
+	log.Infof("searching for cloudfront distribution %s", name)
+
+	input := &cloudfront.ListDistributionsInput{MaxItems: aws.Int64(100)}
+
+	var distribution *cloudfront.DistributionSummary
+	err := c.Service.ListDistributionsPagesWithContext(ctx, input,
+		func(out *cloudfront.ListDistributionsOutput, lastPage bool) bool {
```

Comment:
Pretty cool 👍 I didn't realize it's so tricky to get information about a distribution but I guess all the functions use the `id` and we do it based on the `alias` name